openc3/lib/openc3/io/json_rpc.rb

```diff
@@ -56,11 +56,15 @@ def as_json(options = nil) self end #:nodoc:
 
 class String
   NON_ASCII_PRINTABLE = /[^\x21-\x7e\s]/
-
+  NON_UTF8_PRINTABLE = /[\x00-\x08\x0E-\x1F\x7F]/
   def as_json(options = nil)
     as_utf8 = self.dup.force_encoding('UTF-8')
     if as_utf8.valid_encoding?
-      return as_utf8
+      if as_utf8 =~ NON_UTF8_PRINTABLE
+        return self.to_json_raw_object
+      else
+        return as_utf8
+      end
```

Comment:
Where did you run into this? Raw binary again?

Reply:
Raw binary with just small numbers is what we put into the INST IMAGE